### PR TITLE
Issue 280

### PR DESCRIPTION
**Pull request by johngruen**
@nchapman: sorry about that! give this a try

### CODE REVIEW

File: app/styles/modules/_actions.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @extend .l-flex;
+    clear: both;
+
+    li {
+      @extend .l-flex-horizontal;
+    }
+
+    a {
+    flex: 0 0 $default-grid;
+    height: $default-grid;
```
Comment (pdehaan) [inline app/styles/modules/_actions.scss]: Indenting slightly off here.

File: app/styles/modules/_actions.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  }
+
+  .small & {
+
+    a {
+      float: left;
+      display: block;
+      width: $default-grid;
+      height: $default-grid;
```
Comment (pdehaan) [inline app/styles/modules/_actions.scss]: A-Z

**Comment by pdehaan** [comment]
Apart from the two minor nits from me, this works great for me in FF and Chrome. :clap:

**Comment by nchapman** [comment]
:+1:

**Pull request merged**